src/tracers.jl

```diff
@@ -120,8 +132,8 @@ $SET_TYPE_MESSAGE
 
 For a higher-level interface, refer to [`hessian_pattern`](@ref).
 """
-struct HessianTracer{S} <: AbstractTracer
-    inputs::Dict{UInt64,S}
+struct HessianTracer{S,I<:Integer} <: AbstractTracer
```

Comment:
parametrize Hessian by the dict type in order to allow `SortedDict`s (we can assume it's `<:AbstractDict`)